km62/Shumel_Sofiia/laboratory1.sql

```diff
@@ -1,2 +1,149 @@
 -- LABORATORY WORK 1
 -- BY Shumel_Sofiia
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+створювати таблиці та видаляти з них дані
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER STUDENT IDENTIFIED BY pass
+DEFAULT TABLESPACE "USERS"
+TEMPORARY TABLESPACE "TEMP"
+QUOTA 50M FROM "USERS"
+
+GRANT CREATE ANY TABLE FOR STUDENT_NAME
+GRANT DROP FOR STUDENT_NAME
+
+
+
+
+
+
+
+
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Студент здає роботу викладачу.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+CREATE TABLE student
+(
+  name_student char(30) NOT NULL,
+  id_student char(10) NOT NULL,
+  course_student char(1) IS NULL,
+  id_labs char(10) NOT NULL,
```

Comment:
In relation one to many id of parent should be placed in child table. Since lab table has student id please remove id_labs from student table.